lib/liquid/condition.rb

```diff
@@ -27,10 +27,32 @@ class Condition #:nodoc:
       end,
     }
 
+    class MethodLiteral
+      attr_reader :method_name, :to_s
+
+      def initialize(method_name, to_s)
+        @method_name = method_name
+        @to_s = to_s
+      end
+
+      def to_liquid
```

Comment:
Since those now only appear in binary expressions in conditions, I don't think this method is necessary anymore.